### PR TITLE
do not recalculate an already calculated bool

### PR DESCRIPTION
**Pull request by dirvine**
Small tidy up

---

This change is [Reviewable](https://reviewable.io/reviews/maidsafe/rust-utp/22)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @canndrew

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/socket.rs [hunk 1]
```diff
@@ -1385,7 +1384,7 @@ impl UtpSocket {
         // Three duplicate ACKs mean a fast resend request. Resend the first unacknowledged packet
         // if the incoming packet doesn't have a SACK extension. If it does, the lost packets were
         // already resent.
-        if !self.send_window.is_empty() && self.duplicate_ack_count == 3 &&
+        if packet_loss_detected && self.duplicate_ack_count == 3 &&
```
Comment (vinipsmaker) [inline src/socket.rs]: You might want to remove `&& self.duplicate_ack_count == 3` too.

**Comment by vinipsmaker** [comment]
These stylistic changes add so much noise that damage/slow the review and add a good chance to make [every other PR](https://github.com/maidsafe/rust-utp/pulls) to conflict for no good reason.

**Comment by dirvine** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

_[src/socket.rs, line 1387 [r1]](https://reviewable.io:443/reviews/maidsafe/rust-utp/22#-KECNoC4LFw4Rceuz5Cp-r1-1387:-KECo8fiaCwljGeS_ZLa:-1750477919) ([raw file](https://github.com/maidsafe/rust-utp/blob/4b837b3c3d34c09527a150e5be08d90d88094164/src/socket.rs#L1387)):_
Of course :-) Done.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/rust-utp/22)_

**Pull request closed without merging**